Signal/src/ViewControllers/CallViewController.swift

```diff
@@ -158,7 +159,9 @@ class CallViewController: OWSViewController, CallObserver, CallServiceObserver,
     }
 
     func didBecomeActive() {
-        shouldRemoteVideoControlsBeHidden = false
+        if (self.isViewLoaded) {
+            shouldRemoteVideoControlsBeHidden = false
```

Comment:
We can become active before we load?  That's very unexpected.  Are you sure?

Reply:
Okay, this makes sense.  Sorry!

Reply:
It was my most likely interpretation for accessing a nil view which is created in viewDidLoad.

If you'd like to look into it yourself, it's our top crash.